pkgs/by-name/xo/xonsh/xontribs/xontrib-broot/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  pkgs,
```

Comment:
Generally individual dependencies seem to be split out to make it easier to override them in an overlay or other usage. So instead of `pkgs` here this should probably be split out to
```nix
buildPythonPackage,
fetchFromGitHub,
lib,

setuptools,
pdm-pep517,
poetry-core,
```

Then each of those things should be referenced direct. Similarly for the other files.